src/Components/EnvironmentPicker/EnvironmentPicker.tsx

```diff
@@ -468,7 +468,7 @@ const EnvironmentPicker = ({
         resources: Array<IAzureResource | string>
     ) => {
         dispatch({
```

Comment:
I had left a comment about combining the local storage values into a single "Environment" object. I don't see that here.

Reply:
it is outside of the scope of this original PR. We can further discuss how to keep the data in local storage for a separate PR.